website/src/pages/create/user_reply.tsx

```diff
@@ -8,6 +8,7 @@ import poster from "src/lib/poster";
 import { Messages } from "src/components/Messages";
 import { TwoColumns } from "src/components/TwoColumns";
 import { Button } from "src/components/Button";
+import { LoadingScreen } from "@/components/Loading/LoadingScreen";
```

Comment:
please use `src` instead of `@`

Reply:
I think it's fine if vs code is doing it.  I'm going to delete this mapping from tsconfig soon so it stops happening.